packages/ppx/src/Property_to_runtime.re

```diff
@@ -1765,11 +1755,7 @@ let render_background = (~loc, background: Types.property_background) => {
     ]
     @ (
       switch (position) {
-      | Some((pos, Some(((), size)))) => [
-          [render_background_position(~loc, pos)],
-          [[%expr CSS.backgroundSize([%e render_bg_size(~loc, size)])]],
-        ]
-      | Some((pos, None)) => [[render_background_position(~loc, pos)]]
+      | Some(_) => raise(Unsupported_feature)
```

Comment:
Why? :(

Reply:
dunno why i left this out, should be backk